_includes/header.html

```diff
@@ -1,76 +1,80 @@
 
 <header role="banner" class="site-header">
-
 	<a href="/" class="site-brand">
 		<img src="/resources/brand/logo.svg" alt="" class="site-brand__logo"/>
 		<span class="site-brand__name">Pa11y</span>
 	</a>
-	<div class="site-navigation" aria-label="Main Navigation">
-		<nav role="navigation">
-			<ul class="site-navigation__menu clearfix" tabindex="0" role="menubar" aria-controls="st1" >
 
-				<li class="site-navigation__item {% if page.url == '/' %}site-navigation__item--current{% endif %}" >
-					<a href="/" class="site-navigation__item-link">
+	<div class="site-nav" aria-label="Main Navigation">
+		<nav role="navigation">
+			<ul class="site-nav__menu clearfix" role="menubar" aria-controls="st1" >
+				<li class="site-nav__item">
+					<a href="/" class="site-nav__link {% if page.url == '/' %}site-nav__link--current{% endif %}">
 						Home
 					</a>
 				</li>
 
-				<li class="site-navigation__item {% if page.url contains '/news/' %}site-navigation__item--current{% endif %}" >
-					<a href="/news/" class="site-navigation__item-link">
+				<li class="site-nav__item">
+					<a href="/news/" class="site-nav__link {% if page.url contains '/news/' %}site-nav__link--current{% endif %}">
```

Comment:
might be fancy to add aria-current to the current menu item too ✨ 

https://tink.uk/using-the-aria-current-attribute/